controller/scheduler/fleet.py

```diff
@@ -377,6 +376,7 @@ def attach(self, name):
 
 CONTAINER_TEMPLATE = [
     {"section": "Unit", "name": "Description", "value": "{name}"},
+    {"section": "Service", "name": "EnvironmentFile", "value": "/etc/environment"},
```

Comment:
What are we thinking `/etc/environment` will have that the app units will need? It may be easier for malicious apps to glean environment information if they're sourcing things like `COREOS_PRIVATE_IPV4`.